src/sass/_vars.sass

```diff
@@ -95,6 +95,8 @@ $colorLineDark: $grey30
 $baseFontMultiplier: 1.125
 $baseFontSize: 1em * $baseFontMultiplier // 18px
 
+$fontSize9: 0.5625em / $baseFontMultiplier
```

Comment:
Text this small might pose accessibility issues. May want to run it through the WAVE tool

Reply:
I didn't get any errors with WAVE.

Reply:
Also, I've noticed it looks smaller in Fractal than it does on an actual page.